README.md

```diff
@@ -1 +1,163 @@
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Portfolio-setup-and-mobile-first] <a name="about-project"></a>
+
+**[Portfolio-setup-and-mobile-first]** is my first project with Microverse 😀. Here we'll see a very basic web page using the standards of the industry 💻
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Languages and technologies</summary>
+  <ul>
+    <li><a href="#">HTML</a></li>
+  </ul>
+    <ul>
+    <li><a href="#">CSS3</a></li>
+  </ul>
+    <ul>
+    <li><a href="#">VScode</a></li>
+  </ul>
+    <ul>
+    <li><a href="#">Git and GitHub</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Easy navigation]**
+- **[Personalized product]**
+- **[Secure payment processing]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need GitHub account and a text editor.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+https
+cd my-folder
+git clone https://github.com/J-C-S-V/hello_microverse_project.git
+
+
+### Install
+
+Install this project with:
+
+N/A
+
+### Usage
+
+To run the project, execute the following command:
+
+N/A
+
+### Run tests
+
+N/A
+
+### Deployment
+
+You can deploy this project using:
+
+GitHub pages
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 🙎‍♂️ Author <a name="authors"></a>
+
+🙎‍♂️ **Juan Sanchez**
+
+- GitHub: [@githubhandle](https://github.com/J-C-S-V)
+- Twitter: [@twitterhandle](https://twitter.com/juansan0)
+- LinkedIn: [@LinkedIn](https://www.linkedin.com/in/juan-carlos-sanchez-vargas-a308b014b/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Payment]**
+- [ ] **[Footer]**
+- [ ] **[Profile picture]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project give me a star 🤓
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank Microverse
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] [OPTIONAL] It is giving a `404` error when clicking on the license information. It would be great if you add a license file to the repository. You can use the `MIT.md` file from the [readme template](https://github.com/microverseinc/readme-template) repository and customize it to fit the project. ✍️

Reply:
It happens also with the MIT hyperlink from the Microverse template. It populates 404 error.

Reply:
Yeah, it is happening because of the naming issue. The file is named `MIT.md` while the link is referencing `LICENSE`. You can copy the MIT file to your repo and rename it to LICENSE.

Reply:
Got it. I will try it with the next project, I'm still late on this one.

Thank you!